client/request.go

```
@@ -747,6 +868,16 @@
 	*fasthttp.Args
 }
 
+// Keys method returns all keys in the form data.
+func (f *FormData) Keys() []string {
+	keys := make([]string, f.Len())
+	f.VisitAll(func(key, value []byte) {
+		keys = append(keys, utils.UnsafeString(key))
```

golangci-lint (1.62.2)

append to slice `keys` with non-zero initialized length

(makezero)